packages/documentation-site/patternfly-docs/content/design-guidelines/charts/about/about.md

```diff
@@ -65,3 +65,15 @@ On click
 
 1. **Legend:** When a user clicks on a legend label, it becomes disabled and the color swatch is replaced with an eye-slashed icon.
 2. **Chart:** Data corresponding to the clicked legend label is hidden from view.
+
+## Develop with charts 
+
+Default styles in the [@patternfly/react-charts package](https://www.npmjs.com/package/@patternfly/react-charts) are aligned with our light theme. To use chart styles, you must also have the [@patternfly/patternfly package](https://www.npmjs.com/package/@patternfly/patternfly) installed.
```

Comment:
I'm a little confused about what it means to "use chart styles". Can you elaborate on that a bit? Just so it's clear (it probably is, just want to make sure) - `@patternfly/react-charts` in light theme works just fine without `@patternfly/patternfly` or any CSS from `@patternfly/patternfly`. The styles are all basically hard-coded into the chart SVG's.

Reply:
Ah nope, I was misunderstanding/assuming that you had to install @patternfly/patternfly in either case! This is ringing a bell for me now, ty